launchable/commands/record/session.py

```diff
@@ -72,6 +72,11 @@ def session(
     you should set print_session = False because users don't expect to print session ID to the subset output.
     """
 
+    if "/" in build_name or "%2f" in build_name.lower():
```

Comment:
I think we need this validation only for the build command.

Reply:
fixed
https://github.com/launchableinc/cli/pull/530/commits/1ac7aa7512721010d60cc22f5fe185734a7cce92